package-lock.json

```diff
@@ -119,6 +119,21 @@
         "node": ">=22.11.0"
       }
     },
+    "apps/care_scribe_fe": {
```

Comment:
Are the changes in this file needed for this PR 🤔

Reply:
Whoops forgot to recheck files, thanks